### DISCUSSION TITLE
Educational status proof

### DISCUSSION THREAD

**Discussion by samietadam** [community/community#122800] [GitHub Education]
### Hi everyone,

I gave proof in French to confirm my educational status, will it be approved and accepted?
![school certificate](https://github.com/community/community/assets/97306814/7cbd4138-0cf5-4f9b-a3da-7bc10373cd51)

**Comment by davevad93** [comment]
Hi @samietadam ,

I don't know if GitHub will accept your document because it doesn't show a “real date” I mean, your document only shows the academic year (2023/2024), I say this because when I submitted the request for my student benefits I've uploaded my ID and my enrollment sheet and they have rejected it because it only showed the academic year as in your case, the reason GitHub gave was that it wasn't possible to prove my student status because the document required a "current date" as an evidence that I was still studying in that school. So I uploaded a screenshot of my last exam results that showed a current date and finally they have accepted my request. So in case they reject yours I advise you to do what I did.
